examples/project_Cubico.py

```diff
@@ -0,0 +1,446 @@
+"""
+This is the import script for Cubico's Kelmarsh & Penmanshiel projects. These
+projects are available under a Creative Commons Attribution 4.0 International
+license (CC-BY-4.0), and are cited below:
+
+*Kelmarsh*:
+
+    Plumley, Charlie. (2022). Kelmarsh wind farm data [Data set].
+    Zenodo. https://doi.org/10.5281/zenodo.5841833
+
+*Penmanshiel*:
+
+    Plumley, Charlie. (2022). Penmanshiel Wind Farm Data [Data set].
+    Zenodo. https://doi.org/10.5281/zenodo.5946807
+
+Below is a description of the data imported and an overview of the
+steps taken to correct the raw data for use in the OpenOA code.
+
+1. SCADA
+   - 10-minute SCADA data for each of the turbines in the project
+   - Power, wind speed, wind direction, nacelle position, wind vane, temperature,
+   blade pitch
+2. Meter data
+   - 10-minute performance data provided in energy units (kWh)
+
+3. Curtailment data
+   - 10-minute availabilitya and curtailment data in kwh
```

Comment:
small type here, and I'd suggest making the spacing between numbered items consistent in this list.

Reply:
Done